src/http/ngx_http_mruby_module.c

```diff
@@ -2474,7 +2524,11 @@ static int ngx_http_mruby_ssl_cert_handler(ngx_ssl_conn_t *ssl_conn, void *data)
   mrb = mscf->state->mrb;
   mrb->ud = mscf;
   ai = mrb_gc_arena_save(mrb);
-  mrb_run(mrb, mscf->ssl_handshake_code->proc, mrb_top_self(mrb));
+  if (mscf->ssl_handshake_code != NGX_CONF_UNSET_PTR) {
+    mrb_run(mrb, mscf->ssl_handshake_code->proc, mrb_top_self(mrb));
+  } else {
+    mrb_run(mrb, mscf->ssl_handshake_inline_code->proc, mrb_top_self(mrb));
+  }
```

Comment:
When both `ssl_handshake_code` and `ssl_handshake_inline_code` have byte-code, these both byte-codes should run at this phase.